src/Web/Web.csproj

```diff
@@ -38,7 +35,12 @@
 		<Exec WorkingDirectory="$(SpaRoot)\" Command="npm install" />
 	</Target>
 
-	<Target Name="PublishRunWebpack" BeforeTargets="Build" DependsOnTargets="NpmInstall">
+	<Target Name="NpmRunBuild" AfterTargets="NpmInstall">
```

Comment:
Nu putem aici sa executam asta pentru un anumit launch configuration?

Reply:
Sau sa il pui sa se execute doar pentru Release, nu si Debug cu presupunerea ca dotnet run face cu Release nu Debug build-ul?

Reply:
Pentru un anumit launch configuration nu am gasit cum sa fac asta, iar presupunerea ca ei ruleaza in Release cu dotnet run nu stiu daca e cea mai ok. In principal momentan acel build nu ne ingreuneaza foarte tare, adauga vreo 10 secunde la startup.

Reply:
ok, deci aici lasam pentru toate.